front/app/components/IdeasMap/desktop/IdeaMapOverlay.tsx

```diff
@@ -186,6 +188,14 @@ const IdeaMapOverlay = memo<Props>(
                 // Ref to use to focus on the overlay after selecting an idea
                 ref={overlayRef}
               >
+                <Box display="flex" pt="16px" pl="16px">
+                  <GoBackButton
+                    onClick={() => {
+                      onSelectIdea(null);
+                    }}
+                  />
+                </Box>
```

Comment:
This was missing from the Idea Map Overlay, so I've added back a Go Back Button.

Reply:
PR: https://github.com/CitizenLabDotCo/citizenlab/pull/10750/files

Reply:
Ah gotchya! Missed that - I though it had been removed accidentally. Indeed, I'll remove the change from my PR then :+1: